core/src/main/scala/akka/kafka/ConsumerMessage.scala

```diff
@@ -113,9 +113,6 @@ object ConsumerMessage {
     override def _2: Long = offset
 
     override def canEqual(that: Any): Boolean = that.isInstanceOf[PartitionOffset]
-
-    def copy(key: GroupTopicPartition = this.key, offset: Long = this.offset): PartitionOffset =
-      PartitionOffset(key, offset)
```

Comment:
Why do we have to remove this? I know this is not intended to be user api, but it is still a breaking bin compat change and someone tried to make it compatible previously (probably when changing from case class).

Reply:
Is it required to be binary compatible?

Reply:
I see, ok that is difficult to work around. Let's remove it, as you have done.